modules/relax/handlers/BaseHandler.cfc

```diff
@@ -39,10 +39,14 @@ component {
 		};
 
 		prc.addCSS = function( assetPath ){
```

Comment:
You know, that's a really good question, at this point, considering they are only used in that file. :) I suspect my initial reason for putting them in there was so that additional assets could be loaded outside of the handler inheritance chain.

They aren't necessary now, though. I'll move them to separate methods in the handler and call it good.